### DISCUSSION TITLE
JaversSchemaManager.ensureTable usecase is not handled properly for Postgres database

### DISCUSSION THREAD

**Discussion by PankajKhairnar07** [javers/javers#1246] [Q&A]
I have created the standalone spring boot application with the "Postgres" database. Whenever the server starts the first time all tables are getting appropriately created in the schema used by my application. When I restart the application getting an error as "ERROR: relation "jv_global_id" already exists". 
I checked the code "PgDatabaseMetaData" already skipping the system default namespace. so if we are getting results from the result set means we already have a table that exists in the Postgres database.
**Note**: Using "demo" as schema name but for others, it can be any random string.

![image](https://user-images.githubusercontent.com/12293966/209937099-20f01473-3c7e-4752-9c9d-258975b613ea.png)

Let me know if we are good to create an issue. @BartoszWalacik_twitter  can you please check once

**Comment by bartoszwalacik** [comment]
Hi @PankajKhairnar07 , so you are basically saying that Javers doesn't work with Postgres, because it crashes when launched for the second time?

**Reply by PankajKhairnar07** [reply]
Yes

**Reply by bartoszwalacik** [reply]
let's continue this thread on your PR https://github.com/javers/javers/pull/1248
